datafusion/sql/tests/sql_integration.rs

```diff
@@ -4641,6 +4641,14 @@ fn roundtrip_statement() -> Result<()> {
             group by "Last Name", p.id 
             having count_first_name>5 and count_first_name<10
             order by count_first_name, "Last Name""#,
+            r#"SELECT j1_string as string FROM j1
```

Comment:
Perhaps we should also add a test (as a follow on PR) for `UNION` (not `UNION ALL`)